ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -202,25 +216,45 @@ open class TwoLineTitleView: UIView {
     // MARK: Setup
 
     /// Sets the relevant strings and button styles for the title and subtitle.
+    /// This is a legacy API that maintains Objective C compatibility with older versions of fluentui-apple.
```

Comment:
Should this be marked as deprecated?

Reply:
I think this is more like a convenience initializer, so we probably (?) might not want to get rid of it?

Reply:
This is the "old" way to setup a `TwoLineTitleView`. It only exists here as a temporary measure to maintain backwards compatibility in case any of our clients are using this view. That being said, I don't mind deprecating it if we really want to, especially since we can point people to the new API. It's more a question of how much additional work we want to make for clients upgrading FUA to the next version.

Reply:
If this is only here as a temporary measure, and the plan is to remove it, it should be marked as deprecated (which is not the same as removing it yet! I can show you examples if you need them). If it's expected to remain as a convenience initializer, then the comment should be amended to indicated that it is a known, usable API, and not something that is going to be removed in the future.

Reply:
Simply deprecating this will cause other issues as is. For example, consider this line that's part of `updateSubtitleView` in `NavigationBar`:

```swift
let customTitleView = TwoLineTitleView(style: style == .primary ? .light : .dark)
customTitleView.setup(title: navigationItem.title ?? "", subtitle: subtitle, interactivePart: .title)
```

This causes an issue: **Which version of `TwoLineTitleView.setup` do we use?** As it is, the Swift compiler seems to gravitate towards this shorter version that we're looking to get rid of further down the road. The problem is, there doesn't seem to be a good way to make a function only available to Objective C, which is the ultimate problem since Objective C doesn't support default argument values like Swift does. I've looked [here](https://stackoverflow.com/questions/32384715/how-to-expose-swift-functions-only-to-objc) and [here](https://developer.apple.com/forums/thread/52340), but neither of these solutions worked for me.

The other solution is to get rid of this altogether and force Objective C consumers of FUA's `TwoLineTitleView` to update their constructors, although this is also less than ideal.

Reply:
> TableViewCell has a similar structure where we kept the old API, but added a new API with more parameters, and the old API calls the new API with the values passed into the parameters. If the comment is confusing, we can either remove it or edit it?

Given this, I think we should just remove it.